tests/test_parser/test_complex_outputs.xml

```diff
@@ -109,7 +109,7 @@
             <container classes="cell_output" nb_element="cell_code_output">
                 <container nb_element="mime_bundle">
                     <container mime_type="image/jpeg">
-                        <image candidates="{'*': '_build/jupyter_execute/a4c9580c74dacf6f3316a3bd2e2a347933aa4463834dcf1bb8f20b4fcb476ae1.jpg'}" uri="_build/jupyter_execute/a4c9580c74dacf6f3316a3bd2e2a347933aa4463834dcf1bb8f20b4fcb476ae1.jpg">
+                        <image candidates="{'*': '_build/jupyter_execute/a4c9580c74dacf6f3316a3bd2e2a347933aa4463834dcf1bb8f20b4fcb476ae1.jpg'}" height="400" uri="_build/jupyter_execute/a4c9580c74dacf6f3316a3bd2e2a347933aa4463834dcf1bb8f20b4fcb476ae1.jpg">
```

Comment:
I understand this image gets the `height=400` metadata. However, it seems to end up being css instead of html attribute.

![imatge](https://github.com/executablebooks/MyST-NB/assets/23738400/05ae10fc-2127-45aa-be6e-80a07ba00e19)

From the description and the xml here I expected to get `<img ... height="400px">` instead of the `<img ... style="height: 400px;">` that shows up.

Reply:
Yeah, seems like this one is the only one that should get that, weird that the others do, I’ll check that out.

Regarding CSS: That’s another issue, but it should definitely be fixed, I agree!